raspberrypi4-64.coffee

```diff
@@ -0,0 +1,35 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'raspberrypi4-64'
+	aliases: [ 'raspberrypi4-64' ]
+	name: 'Raspberry Pi 4 (using 64bit OS)'
+	arch: 'aarch64'
+	state: 'experimental'
+
+	instructions: commonImg.instructions
+	gettingStartedLink:
+		windows: 'https://docs.resin.io/raspberrypi3/nodejs/getting-started/#adding-your-first-device'
+		osx: 'https://docs.resin.io/raspberrypi3/nodejs/getting-started/#adding-your-first-device'
```

Comment:
This should point to rpi4 docs section, even if not existing now, rather than point to rpi3 docs

Reply:
Fixed.